frontend/src/components/admin/AdminDashboard.js

```diff
@@ -27,30 +56,104 @@ export class AdminDashboard extends Component {
     downloadMineManagerHistory(this.state.mineNo, { window, document });
   };
 
+  compareMineName = (a, b) => {
+    if (a.mine_name < b.mine_name) {
+      return -1;
+    }
+    if (a.mine_name > b.mine_name) {
+      return 1;
+    }
+    return 0;
+  };
+
```

Comment:
Is this being sorted alphabetically? Looks like there is a JS function that does this, localeCompare()

If that's not the purpose of this function, can you rename to make it more clear whats happening?

Reply:
Thanks for finding that, changed.